server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -78,6 +91,20 @@ private static HttpServer createHttpServer(ServerConfig config, LoopResources lo
                 .maxHeaderSize(config.getMaxHeaderSize()));
     }
 
+    private static BiPredicate<HttpServerRequest, HttpServerResponse> compressableType() {
+        return (request, response) -> {
+            String contentType = response.responseHeaders()
+                .get(HttpHeaderNames.CONTENT_TYPE);
+            return Optional.ofNullable(contentType)
+                .map(COMPRESS_CONTENT_TYPES::contains)
```

Comment:
How come we don't compress all types?

Reply:
The cpu cost to compress already compressed files like images/pdfs is larger than the network gain from compressing them.